kalite/topic_tools/__init__.py

```diff
@@ -45,12 +44,15 @@
 # Globals that can be filled
 TOPICS          = None
 CACHE_VARS.append("TOPICS")
-def get_topic_tree(force=False, annotate=False, channel=settings.CHANNEL):
+def get_topic_tree(force=False, annotate=False, channel=settings.CHANNEL, language="en"):
+    translation.activate(language)
```

Comment:
might be good to wrap this in `lcode_to_django_lang`, unless you can guarantee up the stack that this language code is run through the same function.